.circleci/config.yml

```diff
@@ -74,11 +65,6 @@ workflows:
       - test:
           context:
             - DockerHub
-      - style_check:
```

Comment:
Not sure if removing this is alright or how to resolve adding pronto check for circle ci instead of GH actions.

Reply:
We can make it a required check before allowing the merge in GH.

Reply:
Makes sense 👍 . I don't have permissions for doing that, could you do it for me?

Reply:
I think I've done this correctly!